ast/src/test/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitorTest.java

```diff
@@ -497,4 +497,30 @@ public void testGraphicsFuzzMacrosDueToSwitchMacro() throws Exception {
         + "}").contains(ParseHelper.END_OF_GRAPHICSFUZZ_DEFINES));
   }
 
+  @Test
+  public void testParseAndPrintVoidParameter() throws Exception {
+
+    // For simplicity, we deliberately chuck away "void" in function arguments; this test captures
+    // that intent.
+
+    final String program = ""
+        + "int foo(void)\n"
+        + "{\n"
+        + " return 2;\n"
+        + "}\n"
+        + "void main(void)\n"
+        + "{\n"
+        + "}\n";
```

Comment:
What about calls using void as well? E.g. `foo(void);`

Reply:
Turns out this is already captured by the way the grammar is structured, but I have added a test to demonstrate that.